web/ui/react-app/src/App.css

```diff
@@ -10,6 +10,24 @@ button.classic-ui-btn {
     margin-bottom: 20px;
 }
 
+.checkbox{
```

Comment:
New UI uses reactstrap, which uses bootstrap and both have style definitions for checkboxes. Shouldn't the new UI re-use bootstrap styles for basic elements like checkboxes?
There's also support for cross-OS consistient style - https://getbootstrap.com/docs/4.3/components/forms/#checkboxes-and-radios-1

Reply:
Thanks, I'll check later but definitely sounds better than SVG icons from @fortawesome/react-fontawesome

Reply:
Ah, I didn't see this comment before I posted my entire review (multiple comments at once). But yes, using the native reactstrap styles probably makes way more sense!